cookiecutter.json

```diff
@@ -29,5 +29,8 @@
     "pypi_credentials_id": "{% if cookiecutter.ci|lower == 'azure' %}testpypi-tomtom-dev{% else %}artifactory-pypi{% endif %}",
     "docker_registry_credentials_id": "{% if cookiecutter.ci|lower == 'azure' %}tomtom-docker-registry-bintray{% else %}artifactory-docker{% endif %}",
     "ci_ssh_agent_user": "{% if cookiecutter.ci|lower == 'jenkins' %}ssh_svc_ci{% else %}N/A{% endif %}",
-    "azure_build_definition_id": "{% if cookiecutter.ci|lower == 'azure' %}5{% else %}N/A{% endif %}"
+    "azure_build_definition_id": "{% if cookiecutter.ci|lower == 'azure' %}5{% else %}N/A{% endif %}",
+    "build_command": "python setup.py build",
```

Comment:
Can we apply the same here: https://github.com/tomtom-international/azure-pipeline-templates/blob/master/steps/python/python.build.yml? If not yet then it makes sense to add `"{% if cookiecutter.ci|lower == 'jenkins' %}<value>{% else %}N/A{% endif %}` for now.
Same applies of course to `lintCommand` and `testCommand`.

Reply:
As mentioned in the jsl PR (https://github.com/tomtom-international/jsl/pull/18), I think the best solution would be to use a thin layer on top (eg. Makefile), define an interface (with targets) and let the projects overwrite/define the content of such a target.

Reply:
That could be done as second step and for now just here only make it available for Jenkins context and make Azure Pipelines not available.